cypress/components/action-popover/action-popover.cy.tsx

```diff
@@ -291,11 +301,11 @@ context("Test for ActionPopover component", () => {
     });
 
     it.each([
-      ["Sub Menu 1", 0],
-      ["Sub Menu 2", 1],
+      [subMenuOption[0], 0],
+      [subMenuOption[1], 1],
     ])(
       "should close %s and ActionPopover after press Esc keyboard key",
-      (times) => {
+      (name, times) => {
```

Comment:
I don't think the name parameter is used in this test.

Reply:
It's only used in the title. I've seen a few other files like this so thought it'd be fine.